src/prop_types.js

```diff
@@ -82,6 +82,21 @@ export let VLayoutItemPropTypes = {
 
   // Used internally by VLayout
   _gutterTop: React.PropTypes.string,
-  _gutterBottom: React.PropTypes.string
+  _gutterBottom: React.PropTypes.string,
+
+  style: function (props, propName, componentName) {
+    const DANGEROUS_STYLES = [
+      "padding", "paddingTop", "paddingBottom", "paddingLeft", "paddingRight",
+      "margin", "marginTop", "marginBottom", "marginLeft", "marginRight",
+      "height", "width", "display", "position", "float",
```

Comment:
agreed that it's weird to allow 2 ways to control the same thing. If anything, I'd deprecate the `width` prop and keep it in `style.width` since it's what most people would expect. what do you think?